app/test/helpers/repositories.ts

```diff
@@ -116,3 +116,50 @@ export async function setupConflictedRepo(): Promise<Repository> {
 
   return repo
 }
+
+/**
+ * Setup a repository and create a merge conflict with multiple files
+ *
+ * @returns the new local repository
+ *
+ * The current branch will be 'other-branch' and the merged branch will be
+ * 'master' in your test harness.
+ *
+ * The conflicted file will be 'foo', 'bar', and 'baz'.
+ */
+export async function setupConflictedRepoWithMultipleFiles(): Promise<
+  Repository
+> {
+  const repo = await setupEmptyRepository()
+  const filePaths = [
+    Path.join(repo.path, 'foo'),
+    Path.join(repo.path, 'bar'),
+    Path.join(repo.path, 'baz'),
+  ]
+
+  await FSE.writeFile(filePaths[0], '')
```

Comment:
If we use FSE.writeFileSycn, we can do away with the `async` keyword. Since this is a test, I think we are fine with doing things synchronously. That's actually what's happening anyways. I'm don't have a strong opinion either way, so I'll leave that up to you.

Reply:
> I tried this out and it slowed the test execution by ~30% locally (+.4 seconds) and on CircleCI (+8 seconds). Let's not remove the promises approach.